<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 12px &quot;Noto Sans&quot;, &quot;FreeSans&quot;;
	background: black;
	color: whitesmoke;
}</string>
  </property>
  <widget class="QLineEdit" name="inputLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background: #263238;
	border: 1px solid black;
	border-radius: 2px;
	padding: 2px;
}</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: #263238;
	color: white;
	border: 1px solid black;
	border-radius: 2px;
    min-width: 88px;
	min-height: 25px;
}
QPushButton:default {
	border: 1px solid #607d8b;
}
QPushButton:pressed {
    background: #607d8b;
}
QPushButton:disabled {
    color: gray;
}</string>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: #263238;
	color: white;
	border: 1px solid black;
	border-radius: 2px;
    min-width: 88px;
	min-height: 25px;
}
QPushButton:default {
	border: 1px solid #607d8b;
}
QPushButton:pressed {
    background: #607d8b;
}
QPushButton:disabled {
    color: gray;
}</string>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../rsclist.qrc"/>
 </resources>
 <connections/>
</ui>
